homeassistant/components/faadelays/binary_sensor.py

```diff
@@ -0,0 +1,89 @@
+"""Platform for FAA Delays sensor component."""
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, FAA_BINARY_SENSORS
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up a FAA sensor based on a config entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    binary_sensors = []
+    for kind, attrs in FAA_BINARY_SENSORS.items():
+        name, icon = attrs
+        binary_sensors.append(
+            FAABinarySensor(coordinator, kind, name, icon, entry.entry_id)
+        )
+
+    async_add_entities(binary_sensors, True)
+
+
+class FAABinarySensor(CoordinatorEntity):
```

Comment:
You should still inherit from BinarySensorEntity as well (both).

Reply:
Fixed.